lms/djangoapps/instructor_task/tests/test_integration.py

```diff
@@ -134,8 +147,23 @@ def submit_rescore_one_student_answer(self, instructor, problem_url_name, studen
                                                   InstructorTaskModuleTestCase.problem_location(problem_url_name),
                                                   student)
 
-    def test_rescoring_option_problem(self):
-        """Run rescore scenario on option problem"""
+    @ddt.data(
+        (dict(correct_answer=OPTION_2), 0, 2, 2),
+        (dict(num_inputs=2), 2, 4, 0),
+        (dict(num_inputs=4), 2, 8, 0),
+        (dict(num_responses=4), 2, 4, 0),
+        (dict(num_inputs=2, num_responses=4), 2, 8, 0),
+    )
+    @ddt.unpack
+    def test_rescoring_option_problem(self, problem_edit, new_expected_score, new_expected_max, new_expected_score_4):
```

Comment:
could we give the last variable a more explicit name than `new_expected_score_4`?